clients/admin-ui/src/features/user-management/PasswordManagement.tsx

```diff
@@ -1,39 +1,61 @@
-import { HStack } from "@fidesui/react";
+import {
+  Box,
+  HStack,
+  IconButton,
+  TrashCanSolidIcon,
+  useDisclosure,
+} from "@fidesui/react";
+import React from "react";
+import DeleteUserModal from "user-management/DeleteUserModal";
+import { User } from "user-management/types";
 
 import { useAppSelector } from "~/app/hooks";
 import { selectUser } from "~/features/auth";
-import { CustomTextInput } from "~/features/common/form/inputs";
 import Restrict from "~/features/common/Restrict";
 import { ScopeRegistryEnum } from "~/types/api";
 
 import NewPasswordModal from "./NewPasswordModal";
 import UpdatePasswordModal from "./UpdatePasswordModal";
 import { selectActiveUserId } from "./user-management.slice";
 
-const PasswordManagement = () => {
+interface PasswordManagementProps {
+  user?: User;
+  isNewUser: boolean;
+}
+
+const PasswordManagement: React.FC<PasswordManagementProps> = ({
+  user,
+  isNewUser,
+}) => {
   const activeUserId = useAppSelector(selectActiveUserId);
   const loggedInUser = useAppSelector(selectUser);
-
-  if (!activeUserId) {
-    return (
-      <CustomTextInput
-        name="password"
-        label="Password"
-        placeholder="********"
-        type="password"
-        tooltip="Password must contain at least 8 characters, 1 number, 1 capital letter, 1 lowercase letter, and at least 1 symbol."
-      />
-    );
-  }
+  const deleteModal = useDisclosure();
   const isOwnProfile = loggedInUser ? loggedInUser.id === activeUserId : false;
 
   return (
-    <HStack>
-      {isOwnProfile ? <UpdatePasswordModal id={activeUserId} /> : null}
-      <Restrict scopes={[ScopeRegistry.USER_PASSWORD_RESET]}>
-        <NewPasswordModal id={activeUserId} />
-      </Restrict>
-    </HStack>
+    <Box>
+      {activeUserId ? (
+        <HStack>
+          {isOwnProfile ? <UpdatePasswordModal id={activeUserId} /> : null}
+          <Restrict scopes={[ScopeRegistryEnum.USER_PASSWORD_RESET]}>
+            <NewPasswordModal id={activeUserId} />
+          </Restrict>
+
+          {!isNewUser && user ? (
+            <Box>
```

Comment:
I'd prefer to have this delete button not inside `PasswordManagement` since it doesn't have to do with password management. can we hoist it to be a sibling?

Reply:
Absolutely. This makes a lot more sense to me, thanks!